src/components/RightSidebar/Participants/Participant.vue

```diff
@@ -824,9 +824,9 @@ export default {
 					token: this.token,
 					attendeeId: this.attendeeId,
 				})
-				showSuccess(t('spreed', 'Invitation was sent to {actorId}', { actorId: this.participant.actorId }))
+				showSuccess(t('spreed', 'Invitation was sent to {actorId}', { actorId: this.participant.invitedActorId }))
 			} catch (error) {
-				showError(t('spreed', 'Could not send invitation to {actorId}', { actorId: this.participant.actorId }))
+				showError(t('spreed', 'Could not send invitation to {actorId}', { actorId: this.participant.invitedActorId }))
```

Comment:
If we are showing toast message for resending, why not showing it also when sending for the first time?